feat(android): collapse toolbar menu support

Max87ZA opened the pull request:
Adds support for toolbar's menu to Collapse toolbat
Usage:
Create your Collapse toolbar as usual and then add to your controller:
```
        $.collapsingToolbar.addMenuItem(0,` 'Item title',  false); 
        $.collapsingToolbar.setOnMenuItemClickListener(function(e) {
        Ti.API.info("e.menuItemId: "+e.menuItemId);
            switch (e.menuItemId) {
                case 0:
                   Ti.API.log("Menu 0 clicked");
                    break;
            }
        });

m1ga committed:
feat(android): add keyboard show/hide event

m1ga committed:
docs

m1ga committed:
use Android version

m1ga committed:
move to app event

m1ga committed:
move to app event

m1ga committed:
size return value

m1ga committed:
check for listener

m1ga committed:
remove extra check, simplify if statement

m1ga committed:
some more null checks for safety

m1ga committed:
lint

m1ga committed:
hasListener check

m1ga committed:
check app listener in safearea

Max87ZA committed:
kotlin 1.8.10, ios logging

hansemannn committed:
feat: add spring animations

hansemannn committed:
Merge branch 'master' into feature/ios17-spring-animations

Max87ZA committed:
Merge pull request #1 from tidev/master

merge tidev master into max87 kotlin_1810

Max87ZA committed:
network client fix

Max87ZA committed:
(fix):network client fix

Max87ZA committed:
Merge branch 'master' into kotlin_1810

Max87ZA committed:
Merge pull request #2 from Max87ZA/kotlin_1810

Kotlin 1810

Max87ZA committed:
feat(ios): largeTitleAttributes

Max87ZA committed:
feat(ios): largeTitleAttributes

Max87ZA committed:
feat(ios): largeTitleAttributes

Max87ZA committed:
fix(ios): removed commented code

m1ga committed:
Update android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

Max87ZA committed:
Merge branch 'tidev:master' into master

Max87ZA committed:
Merge branch 'feat(ios)largeTitleAttributes'

Max87ZA committed:
fix(ios): removed setLargeTitleAttributes duplicity
fix(ios): removed bounce from TiAnimation
feat(android): Added collapse_toolbar props

Max87ZA committed:
feat(ios): added logging with timestamp

hansemannn committed:
feat: support multi-scene applications

Max87ZA committed:
feat(ios): largeTitleAttributes added

Max87ZA committed:
Merge remote-tracking branch 'upstream/feature/support-scenes'

Max87ZA committed:
revert changes from hans

m1ga committed:
Merge branch 'master' into 230121_keyboard_hide_event

m1ga committed:
Update apidoc/Titanium/App/App.yml

m1ga committed:
Update apidoc/Titanium/App/App.yml

hansemannn committed:
feat(ios): support multi-scene applications (#13941)

* feat: support multi-scene applications

* fix: add keys to template Info.plist to restore backwards compatibility
# Conflicts:
#	iphone/TitaniumKit/TitaniumKit/Sources/API/TiApp.h
#	iphone/TitaniumKit/TitaniumKit/Sources/API/TiApp.m

Max87ZA committed:
Merge pull request #3 from tidev/master

12.3.0.GA merge

Commit:
feat(ios): add timestamp to console's log
fix(ios): handleUrl fixed

Commit:
feat(ios): feat(ios): add overrideUserInterfaceStyle to Picker

m1ga committed:
Merge branch 'master' into 230121_keyboard_hide_event

Commit:
Merge branch 'tidev_master'

Commit:
Merge branch '230121_keyboard_hide_event'

Max87ZA committed:
feat(android): collapse toolbar menu support

Max87ZA committed:
build: merge 'tidev/master'

m1ga commented:
@Max87ZA can you start from a clean master branch? You branch has other changes so it's not only the collapse toolbar menu

hansemannn committed:
feat: add 12.5.0 changelog

hansemannn committed:
Revert "fix(ios): fix unbalanced view controller transitions causing issues on iOS 16+ (#13586)"

This reverts commit 50814d717910f5b42b6f61972b30f8d10f85a268.

m1ga committed:
feat(android): statusBarColor for Window (#14089)

Co-authored-by: Chris Barber <chris@cb1inc.com>

m1ga committed:
chore(android): forward --sdk in module build process (#14112)

* chore(android): forward --sdk in module build process

* optimize code

hansemannn committed:
fix(android): textfield padding (#13513)

* Revert "Revert "fix(android): textfield padding (#13279)" (#13512)"

This reverts commit 918388a75a98c3f5458d27f6ed035f11883b303e.

* fix(android): fix input value

* reset padding

---------

Co-authored-by: m1ga <migamiga@web.de>
Co-authored-by: Michael Gangolf <m1ga@users.noreply.github.com>

AbdullahFaqeir committed:
fix(ios): pause returns false (#14114)

hansemannn committed:
feat(ios): add “interactiveDismissModeEnabled” API (#14103)

* feat(ios): add “interactiveDismissModeEnabled” API

* chore: add docs

m1ga committed:
feat(android): more log output (#13999)

* feat(android): more log output

* remove if

* change for loop

* Update android/runtime/v8/src/native/modules/APIModule.cpp

Co-authored-by: Chris Barber <chris@cb1inc.com>

---------

Co-authored-by: Chris Barber <chris@cb1inc.com>

hansemannn committed:
feat: add 12.5.0.GA changelog

hansemannn committed:
chore(release): bump version

m1ga committed:
fix(android): fix ListView layout with header and footer only (#14117)

* fix(android): fix ListView layout with header and footer only

* remove log

* Update android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewHolder.java

Co-authored-by: Chris Barber <chris@cb1inc.com>

* rename variable

---------

Co-authored-by: Chris Barber <chris@cb1inc.com>

prashantsaini1 committed:
ListView has empty cells while scrolling up/down (#14119)

* fix(android): use Barrier to set constraints

Using Barrier, potentially avoiding an issue of improper layout cycles (refer video in PR).

* fix(android): list-item shifts to left on having accessoryType field

m1ga committed:
fix: allow ignoreLog to be a single string (#14116)

hansemannn committed:
fix(build): update ioslib to 1.7.38 to fix incremental builds in Xcode 16

prashantsaini1 committed:
Android: marker event was only triggered for first and last visible section items (#14124)

* fix(android): fire marker event for all visible items in range of first to last visible item

* chore: remove package.json files

hansemannn committed:
chore(ios): update compatibility for iOS 18+ and Xcode 16+

hansemannn committed:
chore: update ioslib to 1.7.39 to support Xcode 16 provisioning profile handling

hansemannn committed:
feat(ios): support multi-scene applications (#13941)""

hansemannn committed:
Revert "feat(ios): support multi-scene applications (#13941)"""

This reverts commit 7904769a24468ffbbc6fc8a948886311b4e235d8.

Max87ZA committed:
fix: cmake fix

Max87ZA committed:
Merge branch '12_5_X_CarPlay'

Max87ZA committed:
feat(android): added uploader and downloader to webview

m1ga commented:
@Max87ZA you'll need to make different branches for your PRs :smile: don't work in master as that will add all your fixes into one big PR
